bin/resources/shaders/opengl/tfx_fs.glsl

```diff
@@ -950,6 +950,15 @@ void ps_main()
 #if PS_SHUFFLE
 	uvec4 denorm_c = uvec4(C);
 	uvec2 denorm_TA = uvec2(vec2(TA.xy) * 255.0f + 0.5f);
+#if PS_SHUFFLE_SAME
```

Comment:
I need to double check but I don't think the gl shader looks correct.

Reply:
OGL Shader is fine.